src/PrestaShopBundle/Resources/views/Admin/Components/confirmation_messages.html.twig

```diff
@@ -0,0 +1,33 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% if confirmations|length %}
+  <div class="bootstrap">
+    <div class="alert alert-success" style="display:block;">
```

Comment:
@NeOMakinG Inline style ?

Reply:
Since alerts doesn't have any hidden property inside the uikit, prefer at least using `.d-none` class I think

Reply:
d-none ? pour un `display:block` ?

Reply:
My bad, I think nothing should do the job ? otherwise prefer declaring the dblock inside a stylesheet